src/invoices/Voucher/VendorAddress/VendorAddress.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  KeyValue,
+  Loading,
+  Row,
+} from '@folio/stripes/components';
+import { COUNTRY_LABEL_BY_CODE } from '@folio/stripes-acq-components';
+
+import { useVendor } from './useVendor';
+import css from './VendorAddress.css';
+
+const VendorAddress = ({ address, vendorId }) => {
```

Comment:
```suggestion
const VendorAddress = ({ address = {}, vendorId }) => {
```
and we can get rid of `defaultProps`